game/static/game/css/animations.css

```
@@ -0,0 +1,104 @@
+/* ============================================================
+   Logo Glow
+   ============================================================ */
+@keyframes logoGlow {
```

Stylelint (17.11.1)

[error] Expected keyframe name "logoGlow" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)